src/components/z-status-tag/styles.css

```diff
@@ -16,7 +16,7 @@
 
```

Comment:
why did you change status tag component?

Reply:
In pratica dopo che era stata chiusa la componente z-status-tag la UIX ha richiesto un cambiamento per diminuire il delay dell'animazione. Visto che era un micro-fix lo abbiamo fatto in questo branch piuttosto di aprirne un altro.
In pratica è stato diminuito semplicemente il delay dell'animazione